### PR TITLE
docs: update readme with local setup

### PR DESCRIPTION
**Pull request by Samridhi-98**
**Description**

-  Modified `README.md`
-  Added local setup and updated CLI section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,11 +115,21 @@ You can configure this template by passing different parameters in the Generator
 |securityScheme|Name of the security scheme. Only scheme with X509 and Kafka protocol is supported for now.|No|'mySchemeName'|
 |certFilesDir|Directory where application certificates are located. This parameter is needed when you use X509 security scheme and your cert files are not located in the root of your application.|No|`../not/in/my/app/dir`|
 
+## Setup Locally
```
Comment (derberg) [inline README.md:118]:
```suggestion
## Setup locally
```

lower case to be consisted with other sections

Comment (derberg) [inline README.md:118]: I'm thinking if someone would need it for reasons other than development 🤔 maybe better move as subsection under `Development`? Thoughts?

**Review by derberg — COMMENTED** [review]
Thanks! Just some minor comments from my side

File: README.md (hunk 1 above)
Comment (Samridhi-98) [reply README.md:118]: Make sense I will do it, can you kindly suggest me the subhead for that.

Comment (derberg) [reply README.md:118]: I think the one you have is still ok `Setup locally`

add it above this sentence `For local development, you need different variations of this command`

Comment (Samridhi-98) [reply README.md]: Done ✔

Comment (Samridhi-98) [reply README.md]: Done ✔

File: README.md [hunk 2]
```diff
@@ -113,8 +119,18 @@ You can configure this template by passing different parameters in the Generator
...
+
+**Setup locally**
+
+```bash
+# Run following commands in terminal:
+$ git clone https://github.com/{username}/nodejs-template
+$ cd nodejs-template
+$ npm install
 ```
+**Note:** To test your changes locally refer [this](https://github.com/asyncapi/nodejs-template#cli).
```
Comment (derberg) [inline README.md:133]: sorry, one last
maybe remove note and just add `ag https://bit.ly/asyncapi ./ -o output -p server=production. I know it is duplication but inside the same document, so yeah not a big deal I think

Comment (Samridhi-98) [reply README.md]: No problem, I have made changes as you suggested 😊

**Review by derberg — APPROVED** [review]
🚀 👏🏼

**Comment by derberg** [comment]
@all-contributors please add @Samridhi-98 for docs

**Pull request merged**